components/InfectionMedicalcareprovisionStatus.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div class="InfectionMedicalcareprovisionStatus">
+    <div class="InfectionMedicalcareprovisionStatus-heading">
+      <h3 class="InfectionMedicalcareprovisionStatus-title">
+        {{ $t('感染状況・医療提供体制（サマリ）') }}
+        {{ date }}時点
+      </h3>
+    </div>
+    <div class="InfectionMedicalcareprovisionStatus-Box">
+      <div class="InfectionMedicalcareprovisionStatus-Headline">
+        <app-link
+          to="https://www.fukushihoken.metro.tokyo.lg.jp/hodo/saishin/hassei.html"
+        >
+          {{ $t('感染状況') }}
+        </app-link>
+      </div>
+      <div class="InfectionMedicalcareprovisionStatus-description">
+        {{ $t('新規陽性者')
+        }}<span>{{ statuses.data['新規陽性者'].toLocaleString() }}人</span> /
```

Comment:
```suggestion
        }}<span>{{ statuses.data['新規陽性者'].toLocaleString() }}{{ $t('人') }}</span> /
```

「人」や「件」などの単位は翻訳対象に含めたほうがいいと思うのですが、いかがでしょう？

Reply:
今回翻訳対象に含めず、日本語版以外は単位を消す対応を後の翻訳残課題で対応したいと思います。